samples/function_chaining_custom_status/DurableActivity/__init__.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+def main(name: str) -> str:
+    """Activity function performing a specific step in the chain
+    
+    Parameters
+    ----------
+    trumber : int

```

Comment:
I don't think the parameter name and type match the argument here. Maybe you meant to rename the argument and its type hint?

Reply:
LOL. I initially created it as "number" but since we are waiting for the json changes, I changed it to string. Weird its now "trumber"  - will fix it